WallPaperDev/WallPaperDev/Extensions/UnsplashCellIdentifiable.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UnsplashCellIdentifiable.swift
+//  WallPaperDev
+//
+//  Created by Jackson Ho on 3/19/20.
+//  Copyright © 2020 Stephen Ouyang. All rights reserved.
+//
+
+import Foundation
```

Comment:
Nit - once you import UIKit you will not need to import Foundation AFAIK as it includes foundation